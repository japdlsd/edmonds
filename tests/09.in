50 200
1 5 10
1 18 4
1 21 6
1 22 10
1 25 5
1 27 9
1 30 1
2 20 6
2 21 1
2 26 4
2 28 2
2 33 10
2 37 4
2 38 6
2 46 9
2 50 4
3 13 7
3 17 8
3 21 8
3 22 9
3 28 10
3 32 7
3 33 9
3 43 9
3 46 2
3 50 4
4 5 10
4 14 10
4 23 9
4 27 3
4 38 3
4 48 10
5 8 4
5 11 9
5 14 4
5 18 3
5 41 4
5 42 6
5 48 1
6 10 6
6 19 10
6 32 5
6 36 3
7 9 10
7 15 8
7 23 2
7 30 3
7 35 2
7 40 1
7 42 2
7 46 1
8 14 6
8 19 2
8 20 4
8 23 6
8 35 10
8 40 2
8 44 9
8 46 1
9 13 8
9 16 1
9 19 2
9 24 4
9 26 3
9 28 9
9 35 5
9 43 3
10 15 8
10 25 4
10 26 10
10 31 8
10 35 6
10 42 8
10 44 7
10 46 10
11 12 8
11 13 1
11 24 2
11 34 1
11 37 6
11 41 5
11 44 3
12 18 4
12 38 4
12 48 4
13 16 3
13 23 5
13 25 4
13 27 6
13 45 5
14 19 8
14 25 4
14 30 6
14 45 3
14 50 4
15 33 3
15 34 5
15 41 6
16 26 5
16 28 5
16 32 5
16 38 8
16 45 8
16 46 8
16 48 7
16 50 1
17 20 3
17 23 10
17 31 6
17 32 3
17 49 7
17 50 10
18 28 7
18 36 8
18 39 1
18 42 1
19 26 9
19 50 2
20 21 7
20 40 5
20 45 3
20 49 2
21 29 4
21 32 3
21 46 5
22 23 5
22 26 9
22 35 3
23 27 7
23 35 5
23 37 5
23 40 10
23 48 1
24 30 8
24 32 3
24 37 7
24 43 1
24 46 4
24 47 2
24 50 10
25 27 7
25 32 6
25 33 4
25 45 4
25 46 8
25 50 2
26 27 1
26 33 6
26 42 7
26 43 1
26 47 1
26 48 9
27 28 10
27 31 5
27 33 5
27 35 1
27 41 10
27 43 3
27 44 4
27 48 8
27 50 4
28 31 5
28 46 9
29 35 7
29 39 8
29 43 1
29 45 6
31 33 5
32 38 2
32 41 4
32 46 7
33 34 4
33 40 4
34 38 10
34 43 6
34 47 2
34 49 5
35 36 1
35 41 1
35 44 7
35 45 9
35 46 6
36 44 3
36 46 7
37 38 10
37 39 2
37 40 9
37 44 10
37 45 3
37 47 9
38 41 9
40 49 3
41 46 4
42 43 7
42 44 6
44 50 4
45 48 5
46 47 5
47 48 6
47 50 9
