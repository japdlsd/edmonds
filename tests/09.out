61
44 50
37 39
35 41
34 49
29 43
28 31
26 47
24 32
22 23
18 42
17 20
15 33
14 45
12 38
11 13
10 25
9 16
8 19
7 40
6 36
5 48
4 27
3 46
2 21
1 30

